packages/@lwc/engine-core/src/framework/profiler.ts

```diff
@@ -49,6 +49,27 @@ const operationIdNameMapping = [
     'lwc-rehydrate',
 ] as const;
 
+const operationTooltipMapping = [
+    // constructor
+    'component constructor()',
+    // render
+    'component render() and virtual DOM rendered',
+    // patch
+    'component DOM rendered',
+    // connectedCallback
+    'component connectedCallback()',
+    // renderedCallback
+    'component renderedCallback()',
+    // disconnectedCallback
+    'component disconnectedCallback()',
+    // errorCallback
+    'component errorCallback()',
+    // lwc-hydrate
+    'component first rendered',
+    // lwc-rehydrate
+    'component re-rendered',
+] as const;
```

Comment:
```suggestion
] as const satisfies Record<OperationId, string>;
```
Using `satisfies` ensures we don't forget any IDs in the list.